tools/stop_machine_aws.py

```diff
@@ -1,39 +1,43 @@
 import subprocess
-import paramiko
 import boto3 
 import os
 import sys
+import logging
+import argparse
 
-def stop(instance,f_log):
-    if instance.state['Name']!='stopped':
-            f_log.write("Id: {0}, Platform: {1},Type: {2}, Public IPv4: {3}, AMI: {4}, State: {5}\n".format(instance.id, instance.platform, instance.instance_type, instance.public_ip_address, instance.image.id, instance.state))
+def stop(instance, verbose):
+    if instance.state['Name'] != 'stopped':
+            if verbose:
+                logger.info("Id: {0}, Platform: {1},Type: {2}, Public IPv4: {3}, AMI: {4}, State: {5}\n".format(instance.id, instance.platform, instance.instance_type, instance.public_ip_address, instance.image.id, instance.state))
```

Comment:
No need for if verbose if using the correct log level and log function.
See comment below